Gordon360/Services/RecIM/MatchService.cs

```diff
@@ -0,0 +1,188 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Text.RegularExpressions;
+using Match = Gordon360.Models.CCT.Match;
+
+namespace Gordon360.Services.RecIM
+{
+    public class MatchService : IMatchService
+    {
+        private readonly CCTContext _context;
+
+        public MatchService(CCTContext context)
+        {
+
+            _context = context;
+        }
+        public MatchViewModel GetMatchByID(int matchID)
+        {
+            var match = _context.Match
+                        .Where(m => m.ID == matchID)
+                        .Select(m => new MatchViewModel
+                        {
+                            ID = matchID,
+                            Time = m.Time,
+                            Surface = _context.Surface
+                                        .FirstOrDefault(s => s.ID == m.SurfaceID)
+                                        .Description,
+                            Status = _context.MatchStatus
+                                        .FirstOrDefault(ms => ms.ID == m.StatusID)
+                                        .Description,
+                            Attendance = _context.MatchParticipant
+                                        .Where(mp => mp.MatchID == matchID)
+                                        .Select(mp => new ParticipantViewModel
+                                        {
+                                            ADUserName = _context.ACCOUNT
+                                                        .FirstOrDefault(a => a.account_id == mp.ParticipantID)
+                                                        .AD_Username
+                                        }).AsEnumerable(),
+                            // Team will eventually be handled by TeamService 
+                            Team = m.MatchTeam.Select(mt => new TeamViewModel
```

Comment:
I think we need team members for a particular match as well. In the UI model we have right now, the members are listed for each team with their pictures.